Bor_Trie/Bor_Trie/Program.cs

```diff
@@ -0,0 +1,115 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Bor_Trie

```

Comment:
```suggestion
namespace Trie

```
 Обрати внимание на стайлгайд: названия классов нужно писать в UpperCamelCase